pyproject.toml

```diff
@@ -38,7 +38,7 @@ scikit-learn = "^1.3.1"
 jiwer = "^3.0.3"
 detoxify = "^0.5.1"
 transformers = "4.22.1"
-sagemaker = "^2.197.0"
+sagemaker = "^2.198.0"
```

Comment:
can we go to 199 instead?

Reply:
2.199.0 has moved to even later version of `boto3 >= 1.33.2` [ref]( https://github.com/aws/sagemaker-python-sdk/blob/master/setup.py#L52) .  `s3fs/aiobotocore` still doesn't accommodate this change yet. (supports <1.33.2)